client/payment-methods/test/index.js

```diff
@@ -135,12 +133,18 @@ describe( 'PaymentMethods', () => {
 			name: 'Delete',
 		} );
 		user.click( ccDeleteButton );
-
-		const expectedUpdatedMethodIds = enabledMethodIds.filter(
```

Comment:
I like to avoid conditionals in test - IMO testing with defined constants is more effective than testing with tests also performing conditional logic